BSP/STM32F413/Src/BSP_Timer.c

```diff
@@ -13,9 +16,9 @@ void BSP_Timer_Init(void) {
 	RCC_APB1PeriphClockCmd(RCC_APB1Periph_TIM2, ENABLE);
 	TIM_TimeBaseInitTypeDef Init_TIM2;
 	
-	Init_TIM2.TIM_Prescaler = 1;
-	Init_TIM2.TIM_CounterMode = TIM_CounterMode_Down;
-	Init_TIM2.TIM_Period = 0xFFFF-1;
+	Init_TIM2.TIM_Prescaler = PRESCALER;
+	Init_TIM2.TIM_CounterMode = TIM_CounterMode_Up;
+	Init_TIM2.TIM_Period = 0xFFFF;
```

Comment:
Can you add a comment that explains how this affects the clock timing? It would be good to know what the clock frequency is and the maximum time we can reliably measure (how long it takes for the clock to count down from `0xFFFF` to `0`).